Dockerfile

```diff
@@ -63,29 +64,30 @@ COPY ${NETBOX_PATH} /opt/netbox
 COPY docker/configuration.docker.py /opt/netbox/netbox/netbox/configuration.py
 COPY docker/docker-entrypoint.sh /opt/netbox/docker-entrypoint.sh
 COPY docker/launch-netbox.sh /opt/netbox/launch-netbox.sh
+COPY docker/housekeeping.sh /opt/netbox/housekeeping.sh
 COPY startup_scripts/ /opt/netbox/startup_scripts/
 COPY initializers/ /opt/netbox/initializers/
 COPY configuration/ /etc/netbox/config/
 COPY docker/nginx-unit.json /etc/unit/
 
-WORKDIR /opt/netbox/netbox
+WORKDIR /opt/netbox/
```

Comment:
What was the reason behind the decision to change the workdir?

Reply:
This was changed because of warnings from super-linter.